packages/react-scripts/backpack-addons/babelIncludePrefixes.js

```diff
@@ -15,20 +20,18 @@ const customModuleRegexes = bpkReactScriptsConfig.babelIncludePrefixes
     })
   : [];
 
-// Backpack / saddlebag node module regexes
+// Backpack node module regexes
 const backpackModulesRegex = /node_modules[\\/]bpk-/;
-const saddlebagModulesRegex = /node_modules[\\/]saddlebag-/;
 const scopedBackpackModulesRegex = /node_modules[\\/]@skyscanner[\\/]bpk-/;
-const backpackSinglePackageModulesRegex =
-  /node_modules[\\/]@skyscanner[\\/]backpack-web/;
+const backpackMixinsModulesRegex =
+  /node_modules[\\/]@skyscanner[\\/]backpack-web[\\/]bpk-mixins/;
 
 module.exports = () => {
   return [
     paths.appSrc,
     backpackModulesRegex,
-    saddlebagModulesRegex,
     scopedBackpackModulesRegex,
-    backpackSinglePackageModulesRegex,
+    backpackMixinsModulesRegex,
```

Comment:
Still doubt if including this line here adds any value as there's no JS code in bpk-mixins

Reply:
Ah yes you are right! mixins aren't any JS! Will remove that!